### ISSUE TITLE
Running test.sh -d for Sample API throws an error

### ISSUE THREAD

**Issue by MikeTheCanuck** [hackoregon/backend-examplar-2018#33]
Hi guys, I spent some time catching up on the flurry of enhancements that have landed here, and ran into something I'm not sure if it's by design or not.

I've run `./bin/build.sh -d` and `./bin/start.sh -d` on the freshly-cloned repo and both worked great.  Then out of habit I decided to try `./bin/test.sh -d` and it results in:

```
ERROR: 
        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?

        Supported filenames: docker-compose.yml, docker-compose.yaml
        
Stopping test db container
Error response from daemon: No such container: tests_db_1
Removing test db container
Error: No such container: tests_db_1
```

Is this not expected to work for the Sample, but perhaps only for a real API application?

**Comment by MikeTheCanuck** [comment]
Sorry that I didn't just immediately fix this myself - I'm feeling rusty on Docker and didn't want to leap to big conclusions until it comes back to me.

**Comment by bhgrant8** [comment]
yeah, this did need a bit of updating. the script does appear to be missing a docker-compose file at the minimum. maybe we can condense the steps a bit as well. 

once this works we can add a test or two as well, @kiniadit mentioned an option for these in #26

**Comment by bhgrant8** [comment]
have a branch going that will get the local db container going and add some tests. need to add a little documentation so not ready for pr yet. was going to effectively remove the prod testing for the moment, until having a chance to play around with travis again, figure out a good pattern. a little rusty in that vain.

**Comment by bhgrant8** [comment]
closed as fixed in #35
